components/dashboard/src/Login.tsx

```diff
@@ -126,6 +136,12 @@ export function Login() {
                         </div>
                     </div>
                 </div>
+                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
+                    {`Email address is already in use. Please log in with ${emailTaken.otherUser.authHost}.`}
```

Comment:
**suggestion**: Could we use the alert icon (❗) in front of the text as we do with the garbage collection message and other alert component instances? This can help make it clear to the user that this element is not a graphic affordance already existing there but something is wrong with the current state.